test/unit/plugins/jira.test.js

```diff
@@ -1,14 +1,18 @@
 const assume = require('assume');
+const proxyquire = require('proxyquire');
 const sinon = require('sinon');
-const request = require('request-promise-native');
-
-const JiraPlugin = require('../../../plugins/jira');
+const { json } = require('express');
 
 const sandbox = sinon.createSandbox();
```

Comment:
unrelated, but you should probably be using the standard sinon as a sandbox rather than making one.